src/components/ImportTokenButton.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleProp, StyleSheet, TextStyle } from 'react-native'
+import { AnalyticsEventType } from 'src/analytics/Properties'
+import Times from 'src/icons/Times'
+import { navigateBack } from 'src/navigator/NavigationService'
+import { TopBarIconButton, TopBarTextButton } from 'src/navigator/TopBarButton'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+
+interface Props {
+  onCancel?: () => void
+  style?: StyleProp<TextStyle>
+  eventName?: AnalyticsEventType
+  buttonType?: 'text' | 'icon'
+}
+
+export default function ImportTokenButton({
+  eventName,
+  onCancel,
+  style,
+  buttonType = 'text',
+}: Props) {
+  function onPressCancel() {
```

Comment:
looks like you just copied the CancelButton component. calling the onPress handler "onPressCancel" doesn't make sense if the button isn't a cancel button.

Reply:
by the way, if you find yourself copy/pasting entire files like this, you should probably look for something you can reuse instead, like Alex's suggestion on using a generic RN button